git-day-1/README.md

```diff
@@ -71,7 +72,7 @@ Use the guidelines from the prep section of this week to create a fork.
 - ⏲️ 5 mins
 - A volunteer will need to facilitate this section
 
-A volunteer will need to navigate and let the group guide them in creating a fork as per the steps in the previous section.
+A volunteer will need to navigate and let the group guide them in creating a fork according to the steps in the previous section.
```

Comment:
```suggestion
A volunteer will need to navigate and let the group guide them in creating a fork.
```

You don't know it will always have a previous section. Don't tightly couple content.